commands/dns_cli.py

```diff
@@ -321,15 +321,22 @@ def check_r53(domain_session, project_session, domain, base_domain):
     return cert_arn
 
 
-def lb_domain(project_session: Session, app: str, svc: str, env: str) -> Tuple[str, dict]:
+# Todo: Maybe rename method to something which describes its function?
+def get_elastic_load_balancer_domain_and_configuration(
+    project_session: Session, app: str, svc: str, env: str
+) -> Tuple[str, dict]:
+    def separate_hyphenated_application_environment_and_service(hyphenated_string):
```

Comment:
As it stands this method is just to increase readability and reduce duplication in it's parent method. 

I put it in here so that it's effectively a private method (for now anyway) and doesn't need to be tested directly.